cmd/state-installer/cmd.go

```diff
@@ -47,10 +47,15 @@ type Params struct {
 	activate        *project.Namespaced
 	activateDefault *project.Namespaced
 	showVersion     bool
+	nonInteractive  bool
 }
 
 func newParams() *Params {
-	return &Params{activate: &project.Namespaced{}, activateDefault: &project.Namespaced{}}
+	return &Params{
+		activate:        &project.Namespaced{},
+		activateDefault: &project.Namespaced{},
+		nonInteractive:  !term.IsTerminal(int(os.Stdin.Fd())),
```

Comment:
Have you tested this? I can't recall if these values are preserved through captain. I imagine they should be if the user didn't supply the flag, but probably worth verifying.

Reply:
Yes I have. They are preserved if unspecified.